LondonBritishLibrary/orient/BLorient8192.xml

```diff
@@ -26,279 +26,362 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         <repository ref="INS00001BL"/>
                         <collection>Oriental</collection>
                         <idno>BL Oriental 8192</idno>
-                        <altIdentifier><idno>Strelcyn cat. 56</idno></altIdentifier>
+                        <altIdentifier><idno>Or. 8192</idno></altIdentifier>
+                        <altIdentifier><idno>Strelcyn 56</idno></altIdentifier>
                     </msIdentifier>
-<msContents>
-
-<msItem xml:id="ms_i1">
+<msContents><summary/></msContents>
+<physDesc>
+    <objectDesc form="Codex">
+        <supportDesc>
+            <support>
+                <material key="parchment"/>
+            </support>
+            <extent>
+                <measure unit="leaf">149+1</measure>
+                <measure unit="page" type="blank">2</measure><locus target="#1v #Iv"/>
+            </extent>
+        <condition key="good"><locus from="1" to="20"/> are stained by some liquid, particularly the first few. 
+            <locus from="140"/> have been repaired at the bottom.</condition>
+        </supportDesc>   
+    </objectDesc>
+    <additions>
+        <list> 
+            <item xml:id="e1">
+                <locus target="#Ir"/>
+                <desc type="PurchaseNote">A type-written note on the recto of the back guard leaf.</desc>
+                <q xml:lang="en">This ms. was brought from Abyssinia after the campaign of <date when="1868">1868</date> by 
+                    <persName role="owner" ref="PRS14474JohnHills"><roleName type="title">Sir</roleName>John Hills, 
+                    <roleName type="title">K. C. B.</roleName></persName>, and sold to the Trustees of the 
+                    <placeName ref="INS00001BL">British Museum</placeName> in <date when="1918">1918</date> by his sister, 
+                    <persName role="owner" ref="PRS14475VeronicaHills"><roleName type="title">Mrs</roleName> Lewis Pugh</persName>, 
+                    late of Abermaed, Cardiganshire, for the benefit of
+                    the <placeName ref="INS0000000000000000000000000">British Red Cross Society</placeName>.</q>
```

Comment:
either create ID or remove the tag (you may well remove the tag, we do not need to mark everything up)